app/Platform/RouteServiceProvider.php

```diff
@@ -64,6 +75,12 @@ protected function mapWebRoutes(): void
 
                 Route::get('games', [GameController::class, 'index'])->name('game.index');
 
+                Route::get('hubs', [HubController::class, 'show'])
+                    ->defaults('gameSet', GameSet::CentralHubId)
+                    ->name('hubs.index');
+
+                Route::get('hub/{gameSet}', [HubController::class, 'show'])->name('hub.show');
```

Comment:
Not sure where the game redirect is happening, so this comment it's well placed.

There seems to be some bad data in the hub mappings.
http://localhost:64000/game/19594
Should be 
![image](https://github.com/user-attachments/assets/e013e487-d9ba-4258-bb77-2650eaff61f7)

per the link on http://localhost:64000/game/2809, but it looks like the hub maybe got repurposed?
![image](https://github.com/user-attachments/assets/b1e9e6c3-c754-49ff-ad50-5a7d2dff0b5b)

Reply:
Bad data indeed. I've made some adjustments to the game model in latest to ensure these hub titles are being written correctly. When we run `php artisan ra:sync:game-sets` on deploy, all the bad data should be fixed.